docs/Istio-on-karmada-different-network.md

```diff
@@ -0,0 +1,268 @@
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+EOF
+```
+
+ Run the following command to install istio CRDs on karmada apiserver: 
+
+```bash
+istioctl install
+```
+
+ Karmada apiserver will not deploy a real istiod pod, you should press `ctrl+c` to exit installation when `Processing resources for Istiod`. 
+
+```bash
+✔ Istio core installed                                                           
+- Processing resources for Istiod.  
+```
+
+### Install Istiod on member1
+
+1、Close Karmada's auto-sync feature when labeling a namespace,because I need to label the same namespace `istio-system` in the different clusters differently.
+
+ Export `KUBECONFIG` and switch to `karmada host`: 
+
+```bash
+export KUBECONFIG=$HOME/.kube/karmada.config
+kubectl config use-context karmada-host
+```
+
+Edit `karmada-controller-manager`deployment
+
+```bash
+kubectl edit deployment karmada-controller-manager -n karmada-system
+```
+
+add `--controllers=-namespace,*` in start command
+
+2、Set the member1's network
+
+Export `KUBECONFIG` and switch to `member1`: 
+
+```bash
+export KUBECONFIG="$HOME/.kube/members.config"
+kubectl config use-context member1
+```
+
+Label namespace `istio-system`
+
+```bash
+kubectl label namespace istio-system topology.istio.io/network=network1
+```
+
+3、Install istio control plane
+
+```bash
+cat <<EOF | istioctl install -y -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogFile: /dev/stdout
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: member1
+      network: network1
+EOF
+```
+
+4、Install the east-west gateway in `member1`
+
+```bash
+samples/multicluster/gen-eastwest-gateway.sh --mesh mesh1 --cluster member1 --network network1 | istioctl install -y -f -
+```
+
+5、Expose the control plane and service in `member1`
+
+```bash
+kubectl apply -f samples/multicluster/expose-istiod.yaml -n istio-system
+kubectl apply -f samples/multicluster/expose-services.yaml -n istio-system
+```
+
+### Configure `member2` as a remote cluster
+
+1、Enable API ServerAccess to `member2`
+
+switch to `member2`: 
+
+```bash
+kubectl config use-context member2
+```
+
+Label namespace `istio-system`
+
+```bash
+kubectl label namespace istio-system topology.istio.io/network=network2
+```
+
+Prepare member2 cluster secret
+
+```bash
+istioctl x create-remote-secret --name=member2 > istio-remote-secret-member2.yaml
+```
+
+ Switch to `member1`: 
+
+```bash
+kubectl config use-context member1
+```
+
+Apply istio remote secret
+
+```bash
+kubectl apply -f istio-remote-secret-member2.yaml
+```
+
+2、configure member2 as a remote 
+
+  Switch to `member2`: 
+
+```bash
+kubectl config use-context member2
+```
+
+ Save the address of `member1`’s east-west gateway
+
+```bash
+export DISCOVERY_ADDRESS=$(kubectl -n istio-system get svc istio-eastwestgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')

```

Comment:
It's on my side. There will be the following error

![image](https://user-images.githubusercontent.com/17962021/172750479-06154ac9-f050-408e-8809-496f111dd2fd.png)

Reply:
>
>  Switch to `member2`: 
> kubectl config use-context member2
> Save the address of `member1`’s east-west gateway
> export DISCOVERY_ADDRESS=$(kubectl -n istio-system get svc istio-eastwestgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')

Maybe this command is executed in member1?

Reply:
member1